src/Modules/Booking/src/Booking/Booking/Features/CreateBooking/CreateBookingEndpoint.cs

```diff
@@ -10,7 +11,7 @@ namespace Booking.Booking.Features.CreateBooking;
 public class CreateBookingEndpoint : BaseController
 {
     [HttpPost]
-    [Authorize]
+    [Authorize(AuthenticationSchemes = IdentityServerAuthenticationDefaults.AuthenticationScheme)]
```

Comment:
Please remove `AuthenticationSchemes = IdentityServerAuthenticationDefaults.AuthenticationScheme` from here and configure it globally in `AddJwt` method.